### PR TITLE
Assignment 2

### PR DESCRIPTION
**Pull request by Startarrot**
Hello world has a different font and hero is controlled by arrow keys. Couldn't get color to change in hero class but did in playstate. I have it set to toggle by a Boolean if hello world or hero plays in at the top of playstate.

### CODE REVIEW

**Review by SamBumgardner — CHANGES_REQUESTED** [review]
The final functionality of the program looks correct, though I'd be happy to have the "hello world" and hero objects in the game at the same time (makes grading a bit smoother).

For the internals of the code, we need to figure out how to get the Hero's color to be changed within its own update function. In general, we want our game objects to have total control over themselves whenever possible. Troubleshooting weird behavior in a class becomes much harder if different external classes might be touching internal variables - it's not too hard to track down in our tiny test project, but it can definitely get much harder as the game grows.

Also, a bunch of auxiliary files (the `.vscode` directory & all of the `exports` created during the build process) got committed to the repository as part of this branch. We don't want these to exist in github (the IDE config is only useful to folks who use vscode and the export folder's contents are going to change constantly as people make new builds of the project). Basically, both directories represent some "noise" that distracts from the actual code in the project. 

To fix this, you can delete those files, commit it, then add a `.gitignore` file to the project (outlined in the [git setup](https://github.com/SamBumgardner/Intro-To-Video-Game-Development/blob/master/assignments/Assignment_2.md#git-setup) part of assignment 2). After that's done you're free to add the .vscode directory back (I strongly recommend doing so) and build the project as much as you want—none of these extra files will get pushed up alongside your real code.

To summarize the necessary fixes:
 * Make the color-updating functionality work from within the Hero class.
 * Add an appropriate `.gitignore` and make sure `.vscode` and `export` folders no longer are tracked within this branch (won't be added to the repo when you merge).

**Review by SamBumgardner — APPROVED** [review]
Looks good, thanks for executing the fixes!
